test/commands/generate/model_test.rb

```diff
@@ -33,7 +33,16 @@
         assert_generated_file(original_wd.join('test/fixtures/commands/generate/model/car.rb'), 'lib/testapp/entities/car.rb')
       end
     end
+  end
 
+  describe 'sanitizes application name' do
+    it 'downcases it' do
+      with_temp_dir('TestApp') do |original_wd|
+        command = Hanami::Commands::Generate::Model.new({}, 'car')
+        capture_io { command.start }
+        assert_generated_file(original_wd.join('test/fixtures/commands/generate/model/car.rb'), 'lib/testapp/entities/car.rb')
```

Comment:
I don't know because hanami generate `testapp.rb` db config file. @jodosha what do you think?